### ISSUE TITLE
The documentation for podman is pretty out of date

### ISSUE THREAD

**Issue by AxelSilverdew** [overhangio/tutor#771]
As of Podman 3.0.0, podman has full support for docker-compose directly without the need for podman-compose.

Would it be alright if I submitted a PR updating that documentation and adding some more information? Or is there a different channel I should go through first?

**Comment by regisb** [comment]
Hi @AxelSilverdew, thanks for raising this issue. Have you tried to run Tutor with Podman? Was it successful? If yes, a detailed tutorial would be great indeed.

**Comment by AxelSilverdew** [comment]
Hi @regisb, I did a quick test and I am definitely able to run Tutor with Podman.

However, there's 2 problems I ran into:

1.  Given that Podman runs rootless containers, Caddy can't bind to port 80. The fix for this is to run the entire thing behind a proxy using the instructions detailed [here](https://docs.tutor.overhang.io/tutorials/proxy.html).
2.  The mysql container has problems starting up due to some SELinux related issue. I am currently debugging this and will include the fix for it in my documentation.

Other than these, it looks and performs as expected via Podman. :smile:
